rsa/patterns.yml

```diff
@@ -7,7 +6,7 @@ patterns:
     regex:
       version: 1.1
       pattern: |
-        --BEGIN (?:[A-Z]+ )?PRIVATE KEY--+(\\r|\\n|)+[a-zA-Z0-9+/=\s]+(\\r|\\n|)+--+END (?:[A-Z]+ )?PRIVATE KEY--
+        --BEGIN (?:[A-Z]+ )?PRIVATE KEY--+(\\r|\\n|)+[a-zA-Z0-9\\n+/=\s]+(\\r|\\n|)+--+END (?:[A-Z]+ )?PRIVATE KEY--
```

Comment:
That won't do what you want. The sequence `\\n` in a character class will be interpreted to mean `a literal backslash` and `n`.

You'll need to use a group with alternation `(...|...)` for a multi-character choice, e.g.

```
--BEGIN (?:[A-Z]+ )?PRIVATE KEY--+(\\[nr])*([a-zA-Z0-9+/=\s]|\\[rn])+(\\[rn])*--+END (?:[A-Z]+ )?PRIVATE KEY--
````

I took the liberty of also tidying up the original regex code a bit.

Reply:
I've pushed that change 👍 thanks for the fix

Reply:
Fab, let me test it, and approve if it works as expected. I should get around to that tomorrow - sorry about the delay, but I want to make sure I do it justice and I don't have much free time tomorrow! 🙏

Reply:
@aegilops any idea when this will be merged?

Reply:
I tested the change that day, and it didn't behave as I expected, so I wasn't able to merge immediately. Since then I've not returned to it. Let me see what I can do today